dev/ChangeTracker/PathChangeTracker.h

```diff
@@ -0,0 +1,118 @@
+﻿#pragma once
+#include "IChangeTracker.h"
+#include <EnvironmentManager.h>
+#include <wil/registry.h>
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+    struct PathChangeTracker : public IChangeTracker
+    {
+        PathChangeTracker(std::wstring const& pathPart, EnvironmentManager::Scope scope, PathOperation operation);
+        HRESULT TrackChange(std::function<HRESULT(void)> callBack);
+
+
+    private:
+        EnvironmentManager::Scope m_Scope;
+        std::wstring m_PathPart{};
+        std::wstring KeyName();
```

Comment:
Why is this method not const? Does it change 'this'? Or typo and you meant `...m_KeyName{};` ?